### ISSUE TITLE
Agents of type "organization" do not show relationships on the Agent Activity page

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#4151]
https://arctos.database.museum/info/agentActivity.cfm?agent_id=21314878

Why don't we show relationships for organizations? If there isn't a good reason we don't, can we add them?

**Comment by lin-fred** [comment]
@Jegelewicz can you summarize the information you're not seeing again in this thread? You said something about a difference when you're logged in vs not logged in

@dustymc do you know what is going on here?

**Comment by Jegelewicz** [comment]
Public facing page for Arctos Steering Committee - https://arctos.database.museum/info/agentActivity.cfm?agent_id=21314878

![image](https://user-images.githubusercontent.com/5725767/153302626-8f9f6567-16ba-4d60-92b5-b1c79cd43b78.png)

Editable Agent

![image](https://user-images.githubusercontent.com/5725767/153302720-a08efaf7-6a65-4bca-81e7-0379c8986f3d.png)

I don't see any reason that the related agents shouldn't show just like they do on a "person" page

Teresa Mayfield-Meyer - https://arctos.database.museum/info/agentActivity.cfm?agent_id=21300608

![image](https://user-images.githubusercontent.com/5725767/153302866-06d186d6-af12-4e8a-bbc1-6d513e828dde.png)

**Comment by dustymc** [comment]
The page you linked isn't public. I don't think there's any reason to not add relationships, but there's not really any reason to do so either - the info page is designed primarily to view things which might be behind VPDs, and agent-data isn't one of those things; it's all available from the edit UI.

https://arctos.database.museum/agent/21314878 is public and contains relationships.

**Comment by Jegelewicz** [comment]
OK, but why hide them on the non-public version of that page?

**Comment by Jegelewicz** [comment]
@dustymc I think we figured out what is going on. The Arctos Steering Committee agent was originally a "group" agent, but subsequently changed to an "organization". The relationshsips aren't showing up on the agent activity page just for agents that were changed from group to organization. Is something not happening behind the scenes that allows them to display as an organization rather than a group?

**Comment by dustymc** [comment]
/info/agentActivity isn't showing incoming relationships

**Comment by Jegelewicz** [comment]
Add like the other page https://arctos.database.museum/agent/21314878

**Comment by Jegelewicz** [comment]
Changed query for "to relationships" to match /agent - check back to see if it worked.
